test/sparsedir/sparse.jl

```diff
@@ -856,6 +858,7 @@ let  A = sprand(10,10,0.3), B = sprand(10,10,0.3), CF = rand(10,10), AF = full(A
     @test A .+ 3 == AF .+ 3
     @test 3 .+ A == 3 .+ AF
     @test A .+ B == AF .+ BF
+    @test A + AF == AF + A
```

Comment:
just a thought: for this sort of test, it might be best to do `A + AF == A + A == AF + A == AF + AF`, so that hopefully at least one of those (`AF + AF` probably) can be considered a reliable truth so that this test will fail if `sparse + full` stops working entirely.